openfecwebapp/templates/macros/breadcrumbs.html

```diff
@@ -1,21 +1,19 @@
 {% macro breadcrumbs(title, links) %}
-<header class="page-header page-header--primary">
-  <ul class="breadcrumbs">
-    <li class="breadcrumbs__item"><a href="{{ cms_url }}" class="breadcrumbs__link" rel="Home">Home</a></li>
-    {% for link in links %}
-    <li class="breadcrumbs__item">
-      {% if link[0] != '' %}
-        <span class="breadcrumbs__separator">&rsaquo;</span>
-        <a class="breadcrumbs__link" href="{{ link[0] }}">{{ link[1] }}</a>
-      {% else %}
-        <span>{{ link[1] }}</span>
-      {% endif %}
-    </li>
-    {% endfor %}
-    <li class="breadcrumbs__item breadcrumbs__item--current">
+<ul class="breadcrumbs">
```

Comment:
Just circling back here, Noah and I discussed offline. The other templates are using `page-header` macro which has duplicate breadcrumb code. It's okay to make the change included here, but there is follow up work to remove the duplication.